app/models/user.rb

```diff
@@ -50,6 +49,15 @@ class User < ApplicationRecord
             uniqueness: true,
             notify_safe_email: true
 
+  with_options unless: -> { Flipper.enabled?(:cis2) } do
+    validates :password,
+              presence: true,
+              confirmation: true,
+              length: {
+                within: 6..128
+              }
+  end
+
```

Comment:
Would it make more sense to validate the password if the user didn't come from CIS2? Using the feature flag means that invalid users (without a password) can be created while CIS2 is enabled, and then if we disable the flag they'll be invalid. This might not happen in a production env, but in a dev / testing env it could be a nuisance.

Reply:
Yep I think that's a good idea.

Reply:
I've amended the commit with the logic you've suggested.